data/JohnnyLegault.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Jonathan Legault LMT",
+  "displayStatsPublic":true,
+  "type": "personal",
+  "bio": "Hello, I am a Massage Therapist Ready to take on New Clients.",
+  "avatar": "https://github.com/JohnnyLegault.png",
```

Comment:
```suggestion
  "type": "personal",
  "bio": "Hello, I am a Massage Therapist Ready to take on New Clients.",
 
```

Reply:
We don't need the `display` and `avatar` after the new update so remove both of these lines.

Reply:
Thanks, I just fixed them.